bin/utils/getconfig.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+#  Based on Texas Instruments Incorporated - http://www.ti.com/
+
+import argparse
+import curses
+import cv2
+import logging
+import numpy as np
+import os
+import signal
+import stat
+import sys
+from time import time
+from time import sleep
+import threading
+import yaml
+
+_metrics = {}
+
+
+class GetConfigYaml:
+    """
+    Class used to get parameters from a YAML file
+    Parse the parameters for a YAML file with the TI Edge AI segmentation/detection/classification models
+    Args:
+        model_dir (string): The model directory
+    """
+
+    def __init__(self, model_dir):
+        yaml_params = self.read_file(model_dir)
+        self.params = self.parse_params(yaml_params, model_dir)
+
+    def read_file(self, model_dir):
+        try:
+            with open(model_dir + "param.yaml", "r") as file:
+                return yaml.safe_load(file)
+        except Exception as e:
```

Comment:
This exception is too general, only capture the specific one thrown by either `open` or `safe_load`